about.html

```diff
@@ -0,0 +1,72 @@
+        </section>  
+        <div class="aboutus-page">
+            <section class="content-container">
+                <div class="aboutus-subheader">
+                    <h2 class="aboutus-subheading">Transformation through education</h2>
+                </div>
+                <div class="title">
+                    <h2 class="title__copy">Who we are</h2>
+                </div>
+                <div class="about-row">
+                    <div class="about-col">
+                        <img class="about-image" src="images/about.jpg">
+                    </div>
+                    <div class="about-col">
+                        <h2 class="para-title">World's largest university</h2>
+                        <p class=para-copy>Every individual has the potential to create change, whether in their life, their community, or the world. The transformative power of education is what unlocks that potential. Yet, access to high-quality education has been a privilege of the few. From the tried and true to the leading edge. From “for some” to “for all.” By opening the classroom through online learning, edX empowers millions of learners to unlock their potential and become changemakers.</p>
+                        <a href="" class="hero-btn hero-btn--color-red">Explore now</a>
+                    </div>
+                </div>   
+            </section>
+            <section class="content-container">   
+                <div class="title">
+                    <h2 class="title__copy">Our mission</h2>
+                </div>
+                <div class="about-row">
+                    <div class="about-col image-wrapper">
+                        <img class="about-image" src="images/mission.jpg">
+                    </div>
+                    <div class="about-col">
+                        <h2 class="para-title">To transform education system</h2>
+                        <p class=para-copy>Every individual has the potential to create change, whether in their life, their community, or the world. The transformative power of education is what unlocks that potential. Yet, access to high-quality education has been a privilege of the few. From the tried and true to the leading edge. From “for some” to “for all.” By opening the classroom through online learning, edX empowers millions of learners to unlock their potential and become changemakers.</p>
+                    </div>        
+                 </div>   
+            </section>
+           </div>  
+           <section class="footer">
+            <h4 class="footer__heading">About Us</h4>
+            <p class="footer__copy">Excellence University is a leading Educational institution based out of Delhi. The University has established itself as a high quality education provider<br> with prime focus on holistic learning and imbibing competitive abilities in students.</p>
+            <div class="footer__icons">
+                <i class="fa fa-facebook"</i>
```

Comment:
```suggestion
                <i class="fa fa-facebook"></i>
```

applies to all the open <i> tags